src/index.js

```diff
@@ -37,6 +37,14 @@ server.use(utils.auth.jwtCheck.unless({
   path: ['/', '/name', '/version']
 }))
 
+
+server.use(function crossOrigin(req,res,next){
```

Comment:
restify comes with cors support. Just use that instead of rolling your own.

Reply:
where is that documented? I searched and coudnlt find answers to that?

Reply:
Probably use this middleware: https://github.com/Tabcorp/restify-cors-middleware

Reply:
Ahh, ill take a look at that npm. I was hoping that there was more builtin core. But this probably will work.

Reply:
I believe most plugins got migrated. I can check the core code later to see if there's a specific plugin for it.

Reply:
http://restify.com/docs/4to5/#cors -- looks like the plugin is right.

Reply:
👍 works for me